examples/pxScene2d/README.md

```diff
@@ -73,24 +73,53 @@
     ~~~~
 
 3. Build **externals**:
-    ~~~~
-    cd examples/pxScene2d/external
-    ~~~~
-    For Linux and Mac run:
-    ~~~~
-    ./build.sh
-    ~~~~
-    For Raspberry Pi run:
-    ~~~~
-    ./build_rpi.sh
-    ~~~~
-    For Windows (**Run from inside a Visual Studio Command Prompt**):
-    ~~~~
-    buildWindows.bat
-    ~~~~
-
+    a. Build all externals for use during the pxscene build.
+      ~~~~
+      cd examples/pxScene2d/external
+      ~~~~
+      For Linux and Mac run:
+      ~~~~
+      ./build.sh
+      ~~~~
+      For Raspberry Pi run:
+      ~~~~
+      ./build_rpi.sh
+      ~~~~
+      For Windows (**Run from inside a Visual Studio Command Prompt**):
+      ~~~~
+      buildWindows.bat
+      ~~~~
+  
+    b. For using system libraries for all external libs except node and duktape.
```

Comment:
Please detail that this is for linux and mac only or please provide details for building with Windows.

Reply:
Also, since node and duktape could be built/installed locally (not in externals), the 'except node and duktape' wording should change.